tests/functional/diff/__init__.py

```diff
@@ -99,13 +101,30 @@ def setup_package():
 
     test_project_name_new = project_info['name'] + '_' + uuid.uuid4().hex
 
+    # Let's run the second analysis with different
+    # checkers to have some real difference.
+    codechecker_cfg['checkers'] = ['-d', 'core.StackAddressEscape',
```

Comment:
This here looks tricky on first glance, as `-d`,`-e` order is kept instead of the checker order. We should change this to:

`['-e', 'core.CallAndMessage', '-d', 'core.StackAddressEscape']` so the checker order is kept with the previous code in line 73.